<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QMainWindow" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SUPR गाउ खाने कथा</string>
  </property>
  <property name="windowIcon">
   <iconset resource="screen.qrc">
    <normaloff>:/login.png</normaloff>:/login.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1151</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
font-family : century gothic;
font-size: 24px;
background-color: rgb(0, 136, 255);
}

#signuppage{
border-image: url(:/password.jpg);

}



QFrame
{
background-color: rgb(255, 255, 255);
border-radius : 10px;
}

QPushButton 
{
	background-color: rgb(41, 206, 16);
border-radius : 8px;
color : white;
}

QPushButton:hover
{
background-color: rgb(255, 41, 116);
color :white;
}



QLabel {
color : black;
font: 13pt &quot;Segoe UI&quot;;
}

QLineEdit 
{
background : transparent;
border : 1px solid black;
border-radius : 4px;
}

#welcome{

	font: 700 30pt &quot;Segoe UI&quot;;
}

#supr{
background:transparent;
font:700 40pt;
	color: rgb(255, 255, 255);
}

#start{
	background-color: rgb(7, 127, 255);
color:white;
font: 700 13pt ;
}



</string>
    </property>
    <widget class="QWidget" name="loginpage">
     <property name="styleSheet">
      <string notr="true">*{
font-family : century gothic;
font-size: 24px;
background-color: rgb(0, 136, 255);
}

#loginpage{
	border-image: url(:/password.jpg);
}


QFrame
{
background-color: rgb(255, 255, 255);
border-radius : 10px;

}

QPushButton 
{
	background-color: rgb(41, 206, 16);
border-radius : 8px;
color : white;
}

QPushButton:hover
{
background-color: rgb(255, 41, 116);
color :white;
}



QLabel {
color : black;
font: 13pt &quot;Segoe UI&quot;;
}

QLineEdit 
{
background : transparent;
border : 1px solid black;
border-radius : 4px;
}

#pushButton_2:hover
{
color :rgb(255, 41, 116);
}



</string>
     </property>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>491</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 15pt ;
color : black;</string>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>300</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt ;</string>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="login_email">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="login_pw">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>220</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>190</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>350</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Forgot Password?</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>410</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
font: 17px &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Need an account?</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>410</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
text-decoration: underline;
font: 12pt;
color : black;
font: 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>SIGN UP</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
font: 700 9.5pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Database Status...</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="signuppage">
     <property name="styleSheet">
      <string notr="true">#signuppage
{
	
	border-image: url(:/password.jpg);
}</string>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>491</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 15pt ;
color : black;</string>
       </property>
       <property name="text">
        <string>SIGN UP</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>300</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt ;</string>
       </property>
       <property name="text">
        <string>SIGN UP</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>220</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>190</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>380</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
font: 17px &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Already have account?</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>380</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
text-decoration: underline;
font: 12pt;
color : black;
font: 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
font: 700 9.5pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Database Status...</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="welcome_2">
     <property name="styleSheet">
      <string notr="true">#welcome_2{
	
	border-image: url(:/nepal-screen.png);
}

#start
{
border : 1px solid white;
border-radius : 5px;
}

#start:hover
{
background-color: rgb(255, 41, 116);
color :white;
}</string>
     </property>
     <widget class="QLabel" name="welcome">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>210</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;WELCOME TO&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="supr">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>290</y>
        <width>831</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&amp;quot; SUPR गाउ खाने कथा &amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>520</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="screen.qrc"/>
 </resources>
 <connections/>
</ui>
